README.md

```diff
@@ -114,7 +115,7 @@ To do this, set the value of the `dbt_metrics_calendar_model` variable in your `
 config-version: 2
 [...]
 vars:
-    dbt_metrics_calendar_model: ref('my_custom_table')
+    dbt_metrics_calendar_model: my_custom_table # BREAKING change: just the model name, no ref
```

Comment:
This does make [this](https://github.com/dbt-labs/dbt_metrics/pull/11/files#diff-e5abb38c62dae4520103e77b127db027ada56ced396aef708b618d5ed90d2caeR25) _very_ ergonomic, but it would be an outlier in terms of how all other refs are handled (cf [relationships tests](https://docs.getdbt.com/reference/resource-properties/tests#relationships), [other packages](https://github.com/fivetran/dbt_hubspot_source/blob/main/dbt_project.yml#L15) and [metrics themselves](https://docs.getdbt.com/docs/building-a-dbt-project/metrics#declaring-a-metric)) so I would prefer it to stay as a ref instead of an outlier if possible